_posts/en/newsletters/2019-05-07-newsletter.md

```diff
@@ -0,0 +1,273 @@
+---
+title: 'Bitcoin Optech Newsletter #45'
+permalink: /en/newsletters/2019/05/07/
+name: 2019-05-07-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter highlights some of the changes in the
+recently-released Bitcoin Core 0.18.0, describes how bech32 addresses
+are forward compatible with expected protocol improvements, and
+summarizes notable changes in popular Bitcoin infrastructure projects.
+
+## Action items
+
+- **Consider upgrading to Bitcoin Core 0.18.0:** released last week, the
+  latest version of Bitcoin Core brings new features, performance
+  improvements, and bug fixes.  See the *news* section below for
+  more details.
+
+## News
+
+- **Bitcoin Core 0.18.0 released:** this new major version contains
+  several significant new features plus many minor features and
+  bugfixes.  The project's [release notes][0.18 notes] contains a description of
+  all notable changes and a list of each of the 119 people whose merged
+  PRs helped contribute to this release.  Some features we think readers
+  of this newsletter might find especially interesting include,
+
+    - *More PSBT tools and refinements:* the previous major release,
+      0.17, introduced support for [BIP174][] Partially-Signed Bitcoin
+      Transactions (PSBTs) designed to help multiple programs or devices
+      collaboratively create and sign transactions, such as multisig
+      wallets, hardware wallets, and cold wallets.  The 0.18 release
+      builds on that foundation with several bugfixes and improvements,
+      including three new RPCs: `joinpsbts` to merge together multiple
+      PSBTs; `analyzepsbt` to tell the user what they next need to do
+      with the PSBT; and `utxoupdatepsbt` to add necessary information
+      to a PSBT from a node's UTXO set.  Additionally, new [PSBT
+      documentation][] is provided---it's based on the PSBT section from
+      the 0.17 release notes but has been expanded to cover new
+      features.
+
+    - *Initial RPC support for output script descriptors:* Bitcoin
+      wallets need a way to find all the transactions on the block chain
+      that pay their keys.  This is easy if the wallet only supports one
```

Comment:
nit: how about 'pay to addresses controlled by their keys' or 'pay to outputs controlled by their keys' rather than 'pay their keys'?

Reply:
I changed s/their keys/the user/ because I don't think this needs to be too specific given that the next sentence gets into details about scriptPubKey.  However, any of these versions is fine by me if you want to change it.  (I slightly prefer "outputs controlled by their keys" since we later talk about P2PK, which isn't strictly an address.)

Reply:
Eh, on doing a final proofread, I changed this even more.  In any case, please feel free to edit as you see fit at merge time.